samza-core/src/main/java/org/apache/samza/processor/StreamProcessor.java

```diff
@@ -206,16 +219,18 @@ public void onJobModelExpired() {
           if (SamzaContainerStatus.NOT_STARTED.equals(status) || SamzaContainerStatus.STARTED.equals(status)) {
             boolean shutdownComplete = false;
             try {
-              LOGGER.info("Shutting down container in onJobModelExpired.");
+              LOGGER.info("Shutting down container in onJobModelExpired for processor:" + pid);
               container.pause();
```

Comment:
Not sure if we need it as a metric (too many of them anyway). But may be a good addition for debugging. Feel free to add a jira (and implement it if you want).